tasks/task383_matres_classification.json

```diff
@@ -0,0 +1,11564 @@
+{
+    "Contributors": [
+        "Yeganeh Kordi"
+    ],
+    "Source": [
+        "MATRES (http://cogcomp.org/papers/NingWuRo18.pdf), (https://github.com/CogComp/MATRES)"
+    ],
+    "Categories": [
+        "Classification" 
+    ],
+    "Definition": "You will be given a question and a context separated with a \n, and you have to answer that the given verb can be anchored in time or not. We say a verb can be anchored in the real timeline if and only if an verb happened in the past or is happening now or is unconditionally planned to happen in the future. If the verb is not anchorable, provide more detail by choosing one of the other labels. If verb is about a intention, wish or opinion in the future but it's not unconditionaluse No. It's intention/wish/opinion, if verb implies a condition or hypothetical choose No. It's hypothetical/condition this verbs usually come after if, choose No. It's negation if verb is about an event that is not going to happen, if the verb doesn't limited to a specific time choose No. It's abstract/non-specific, If a verb always happens use No. It's static and if it's about a repetitive event use No. It's recurrent. And then classify your answers into \"yes_its_anchorable\", \"no_its_intentionwishopinion\", \"no_its_hypotheticalcondition\", \"no_its_negation\", \"no_its_abstractnonspecific\", \"no_its_static\" and \"no_its_recurrent\".",
```

Comment:
typo: "unconditionaluse"

Reply:
Would it be easier to read if you revise 
> .... No. It's negation. 

 to 
> .... "No, It's negation" ....

?

Reply:
Thanks! I'll fix them now.